splunk-devops-extend/src/main/java/com/splunk/splunkjenkins/console/TeeBuildListener.java

```diff
@@ -0,0 +1,68 @@
+package com.splunk.splunkjenkins.console;
+
+import com.splunk.splunkjenkins.TeeConsoleLogFilter;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import hudson.CloseProofOutputStream;
+import hudson.model.BuildListener;
+import hudson.model.TaskListener;
+import hudson.remoting.RemoteOutputStream;
+import org.jenkinsci.remoting.SerializableOnlyOverRemoting;
+
+import javax.annotation.Nonnull;
+import java.io.Closeable;
+import java.io.FilterOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.PrintStream;
+
+/*
+Helper class that delegates BuildListener functionality to underlying TaskListener
+ */
+public class TeeBuildListener implements BuildListener, Closeable, SerializableOnlyOverRemoting {
+    private final OutputStream out;
```

Comment:
Can you make this a transient field. This will also clear spotbugs warning at lines 31, 37

Reply:
Updated. And added some FB warning suppressions, as we're handling the re-instantiation of fields using the `Replacement` class anyways.